src/components/VisualQueryEditor.tsx

```diff
@@ -95,6 +95,9 @@ export const VisualQueryEditor: React.FC<Props> = (props) => {
   const columns = useColumnOptions(tableSchema.value);
   const groupable = useGroupableColumns(columns);
 
+  const columnTooltip =
+    "Some columns may not be visible for selection. This is due to the type of the column. The visual query editor does not currently support columns of type 'dynamic'";
```

Comment:
```suggestion
    "Some columns may not be visible for selection. The visual query editor does not currently support columns of type 'dynamic'";
```

(the shorter, the better for tooltips)